src/utils/wallet.js

```diff
@@ -11,18 +11,20 @@ export const WALLET_CREATE_NEW_ACCOUNT_URL = 'create'
 export const WALLET_CREATE_NEW_ACCOUNT_FLOW_URLS = ['create', 'set-recovery', 'setup-seed-phrase', 'recover-account', 'recover-seed-phrase']
 export const WALLET_LOGIN_URL = 'login'
 
+export const IS_MAINNET = process.env.REACT_APP_IS_MAINNET || false
```

Comment:
@Patrick1904 "false" will be treated as true here, best to check whether `process.env.REACT_APP_IS_MAINNET` is either "true" or "yes".

Reply:
@vgrichina please check my [update](https://github.com/nearprotocol/near-wallet/pull/500/commits/5a16b6f20a99fb20facdb6c1ed13e4b3968103a0)